packages/core/jest.config.js

```diff
@@ -6,195 +6,33 @@
-  // globalTeardown: undefined,
-
-  // A set of global variables that need to be available in all test environments
   globals: {
     extensionsToTreatAsEsm: [".ts", ".tsx", ".js", ".jsx"],
     "ts-jest": {
       useESM: true
     }
   },
-
-  // The maximum amount of workers used to run your tests. Can be specified as % or a number. E.g. maxWorkers: 10% will use 10% of your CPU amount + 1 as the maximum worker number. maxWorkers: 2 will use a maximum of 2 workers.
-  // maxWorkers: "50%",
-
-  // An array of directory names to be searched recursively up from the requiring module's location
-  // moduleDirectories: [
-  //   "node_modules"
-  // ],
-
-  // An array of file extensions your modules use
   moduleFileExtensions: ["js", "json", "jsx", "ts", "tsx", "node", "mdx"],
-
-  // A map from regular expressions to module names or to arrays of module names that allow to stub out resources with a single module
   moduleNameMapper: {
     "\\.(css|less|sass|scss)$": "identity-obj-proxy"
   },
-
   modulePaths: ["<rootDir>/src"],
-
-  // An array of regexp pattern strings, matched against all module paths before considered 'visible' to the module loader
-  // modulePathIgnorePatterns: [],
-
-  // Activates notifications for test results
-  // notify: false,
-
-  // An enum that specifies notification mode. Requires { notify: true }
-  // notifyMode: "failure-change",
-
-  // A preset that is used as a base for Jest's configuration
   preset: "ts-jest/presets/default",
-
-  // Run tests from one or more projects
-  // projects: undefined,
-
-  // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
-
-  // Automatically reset mock state between every test
-  // resetMocks: false,
-
-  // Reset the module registry before running each individual test
-  // resetModules: false,
-
-  // A path to a custom resolver
-  // resolver: undefined,
-
-  // Automatically restore mock state between every test
-  // restoreMocks: false,
-
-  // The root directory that Jest should scan for tests and modules within
-  // rootDir: undefined,
-
-  // A list of paths to directories that Jest should use to search for files in
   roots: ["<rootDir>"],
-
-  // Allows you to use a custom runner instead of Jest's default test runner
-  // runner: "jest-runner",
-
-  // The paths to modules that run some code to configure or set up the testing environment before each test
   setupFiles: ["<rootDir>/jest.init.js"],
-
-  // A list of paths to modules that run some code to configure or set up the testing framework before each test
-  setupFilesAfterEnv: ["<rootDir>/jest/jest.setup.js"],
-
-  // The number of seconds after which a test is considered as slow and reported as such in the results.
-  // slowTestThreshold: 5,
-
-  // A list of paths to snapshot serializer modules Jest should use for snapshot testing
+  setupFilesAfterEnv: ["<rootDir>/setupTests.js"],
```

Comment:
are you sure it should be running for every test? can it run globally in the global setup?

Reply:
I see you can also do:
```
testEnvironment: 'jest-environment-jsdom',
```
https://testing-library.com/docs/react-testing-library/setup/#jest-27

Reply:
instead of having this file

Reply:
Looks like setupTest is still needed, but changed to jest-environment-jsdom anyway